app/provider/language/actions.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ *
+ * LanguageProvider actions
+ *
+ */
+
+import { CHANGE_LOCALE } from './constants'
+
+export default function changeLocale(languageLocale) {
+  return {
+    type: CHANGE_LOCALE,
```

Comment:
I know this probably hasn't been mentioned, but could you please follow the [ducks convention](https://github.com/erikras/ducks-modular-redux) and co-locate actions, action creators, reducer and selectors in a single file (in this case, `language.duck.ts`)?